_includes/components/2-col.html

```diff
@@ -1,5 +1,5 @@
-<div class="container container--mod {{ include.class }} page-content__prose">
-  <div class="one-account page-content__prose tablet:grid-row">
+<div class="container container--mod {{ include.class }}">
```

Comment:
would it be worth documenting include params while we're we're here?
```suggestion
{% comment %}
includes
- class
- col1
- bannertext
{% endcomment %}
<div class="container container--mod {{ include.class }}">
```

Reply:
> would it be worth documenting include params while we're we're here?

Makes sense! Although honestly this layout is very misleading in presenting itself as something that can be reused, since it [hard-codes images from the "What is Login.gov?" page](https://login.gov/what-is-login/) 😬 

https://github.com/GSA-TTS/identity-site/blob/6d708f004c965e307c3bc7bf4c1b1858663e6abb/_includes/components/2-col.html#L8

https://github.com/GSA-TTS/identity-site/blob/6d708f004c965e307c3bc7bf4c1b1858663e6abb/_includes/components/2-col.html#L25

https://github.com/GSA-TTS/identity-site/blob/6d708f004c965e307c3bc7bf4c1b1858663e6abb/_includes/components/2-col.html#L34

https://github.com/GSA-TTS/identity-site/blob/6d708f004c965e307c3bc7bf4c1b1858663e6abb/_includes/components/2-col.html#L46

Reply:
oof

Reply:
Updated to add comment in 7429bd7.